docs/retries.md

```diff
@@ -63,6 +63,7 @@ access to the following variables:
 - `lastRetry.exitCode`: The exit code of the last retry, or "-1" if not available
 - `lastRetry.status`: The phase of the last retry: Error, Failed
 - `lastRetry.duration`: The duration of the last retry, in seconds
+- `lastRetry.message`: The message output from the last retry
```

Comment:
Ah the fun part of the docs is they aren't versioned. So we'll have to put some sort of "3.5+" message onto this.

Reply:
I've added `(available from version 3.5)` to this